modules/core/src/test/scala/scalacache/memoization/CacheKeySpecCommon.scala

```diff
@@ -50,7 +50,7 @@ trait CacheKeySpecCommon extends Suite with Matchers with BeforeAndAfter {
 
 }
 
-class AClass[F[_]](implicit cache: Cache[F, Int]) {
+class AClass[F[_]]()(implicit cache: Cache[F, Int]) {
```

Comment:
Added the extra `()` here just so they're included in the scala 3 generated cache keys (otherwise `CacheKeyIncludingConstructorParamsSpec."work for a method inside a class"` fails as it would generate `"scalacache.memoization.AClass(${cache.toString}).insideClass(1)"` rather than `"scalacache.memoization.AClass()(${cache.toString}).insideClass(1)"`)